app/src/App/App.css

```diff
@@ -111,3 +111,7 @@ svg {
 .custom-table td {
   color: rgba(0, 0, 0, 0.65) !important;
 }
+
+.main-content__header .ant-btn-primary {
+  margin-right: 0px;
+}
```

Comment:
my thoughts on this:
- the styles defined and exported from within the component should only be those that apply to ui in this package *Only*.
- for the common styles, we should ideally extract those to a central styles file, somewhere like in app or rather in react-utils.
- this will hopefully prepare us towards resolving #356

Reply:
Also we could standardize some of the requested changes in this pr to other related packages i.e for instance this change:
- Remove the extra space below the page title

Reply:
@p-netm created a common style sheet in react utils, to be used in packages where needed.